### DISCUSSION TITLE
Wrong content redirection issue for collaborator and content creator

### DISCUSSION THREAD

**Discussion by sachinb80** [Sunbird-Ed/Community#721] [General Discussion]
Hello, we're using the Sunbird 5.0 version.
Whenever a collaborator adds multiple pieces of content to the same course unit, the material is incorrectly misdirected for both the collaborator and the content author.

A collaborator adding a unit to the course and adding multiple pieces of content within that unit will cause the unit to redirect to another piece of content when the collaborator clicks on it.
And this is taking place for both content creators and collaborators. It functions well for the remaining users.
For reference, kindly watch the attached videos.
Kindly provide a fix or workaround for the same.
@vinukumar-vs @rajeevsathish @pallakartheekreddy

https://github.com/Sunbird-Ed/Community/assets/109504832/8fc53d6f-6ebd-4e34-9b67-f8e0e4cf1993

**Comment by ashokreddy1208** [comment]
@sachinb80 We are able to reproduce this issue on Sunbird ED env for one course, we will discuss internally and back to you on this issue soon.

cc @pallakartheekreddy @vinukumar-vs @rajeevsathish

**Reply by sachinb80** [reply]
okay @ashokreddy1208
